api_tests/lib/ledgers/lnd_instance.ts

```diff
@@ -132,28 +154,35 @@ export class LndInstance {
         );
     }
 
-    public getGrpcSocket() {
-        return `${this.getGrpcHost()}:${this.getGrpcPort()}`;
+    get grpcSocket() {
+        return `${this.grpcHost}:${this.grpcPort}`;
     }
 
-    public getGrpcHost() {
+    get grpcHost() {
         return "127.0.0.1";
     }
 
-    public getGrpcPort() {
-        return this.actorConfig.lndRpcPort;
+    get grpcPort() {
+        return this.lndRpcPort;
     }
 
-    public getLightningSocket() {
-        return `${this.getLightningHost()}:${this.getLightningPort()}`;
+    get p2pSocket() {
```

Comment:
```suggestion
    get lightningSocket() {
```
Best to keep same terminology everywhere (see `lightningHost` and `lightningPort` below)